### ISSUE TITLE
After installing cg3d-maya-casc, Cascadeur hasn't been successfully installed...

### ISSUE THREAD

**Issue by hanna9413** [Nathanieljla/cg3d-maya-casc#2]
Hello, sir! May I ask, after I installed it, Maya already has the "Cascadeur" option. However, in the Cascadeur software, there is no option for Maya, and the "Cascadeur—commands" only has "Reload scripts-Reload commands-Delete objects" three options, the others are missing. Is there any way to handle this?

**Comment by Nathanieljla** [comment]
Yeah.  I'll be home shortly to write a longer response.  Are you on Windows?

**Comment by Nathanieljla** [comment]
I replied to your comment on youtube with a video of how to fix your issue.  Feel free to continue the conversation there if things aren't working out. Others might benefit from the info.

**Comment by Nathanieljla** [comment]
hi Hanna, did you ever get things working?  If so, I'll close this issue.
